packages/apps/immutable-records/package.json

```diff
@@ -39,6 +47,8 @@
     "@vanilla-extract/next-plugin": "2.3.1",
     "eslint": "^8.45.0",
     "eslint-config-next": "13.4.5",
+    "lokijs": "^1.5.12",
+    "pino-pretty": "^10.2.0",
```

Comment:
What is this dependency being used for?

Reply:
see comment on `lokijs`, these two have the same reason.